### ISSUE TITLE
2022-05-18 Patch Releases Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1572]
# Patch Releases

#### Bug fix
- #1440
- #1536
- #1560

#### Accessibility, performance, or security improvement
- #1485

#### Critical institutional link or brand change
- #1547

#### Drupal core - Patch level release (non-security bug-fix release)
- #1553

#### Drupal contrib projects - Add new module
- #1503

#### Other
- #1383
- #1506
- #1300
- #1562

## 2.1.12

Drop support for **2.1.x** with **final patch release 2.1.10**
**Sites are urged to upgrade to 2.2.9 or later immediately.**

#### Bug fix
- #1440
- #1536

#### Accessibility, performance, or security improvement
- #1485

#### Critical institutional link or brand change
- #1547

#### Drupal core - Patch level release (non-security bug-fix release)
- #1553

#### Drupal contrib projects - Add new module
- #1503

#### Other
- #1383
- #1506
- #1300
- #1562

**Comment by joeparsons** [comment]
## 2.2.9 pick list
```
git cherry-pick 1065f080506e90c5acf5c80f810fe29f6483e937
git cherry-pick 9e2dd730f04dd754536834cddb60e80d1113c530
git cherry-pick 0545d3d461573375c0c0780919465b029787ccdc
git cherry-pick 1ba626653aeb566ab712afac997c25de1c8da06b
git cherry-pick ba6b58469c714902910c7d2afbaa7f5089899c8c
git cherry-pick e3ef28e9861e429f0e73b07444d66c065d0f5c57
git cherry-pick 6557271ea4378ad27fa45716ae589b754dccff11
git cherry-pick a25a78b16ff9fbe6346eb567a64aa68a08d3b72c
git cherry-pick dd488491ae5c6cfa137372b7ba1dbe8acd728d52
```
(#1560 should also be merged before creating 2.2.9 release)


## 2.1.10 pick list
```
git cherry-pick 1065f080506e90c5acf5c80f810fe29f6483e937
git cherry-pick 9e2dd730f04dd754536834cddb60e80d1113c530
git cherry-pick 0545d3d461573375c0c0780919465b029787ccdc
git cherry-pick 1ba626653aeb566ab712afac997c25de1c8da06b
git cherry-pick ba6b58469c714902910c7d2afbaa7f5089899c8c
git cherry-pick e3ef28e9861e429f0e73b07444d66c065d0f5c57
git cherry-pick 6557271ea4378ad27fa45716ae589b754dccff11
git cherry-pick a25a78b16ff9fbe6346eb567a64aa68a08d3b72c
git cherry-pick dd488491ae5c6cfa137372b7ba1dbe8acd728d52
```

**Comment by joeparsons** [comment]
## 2.1.10 Release notes snippet
`git log 2.1.9..HEAD --pretty='format:- %s' `

```
- Update drupal/core-recommended requirement from 9.2.18 to 9.2.19 (#1553)
- Closes #1299: Adding documentation surrounding developing and applying updates to quickstart sites. (#1300)
- Update Arizona Bootstrap to 2.0.14 (#1547)
- Fixes #1535 Allow subthemes to override paragraphs and alphabetical listing. (#1536)
- Closes #1505: Enable errorexit for ProboCI migration tests. (#1506)
- Closes #1501: Add environment_indicator module. (#1503)
- Add info about PHPStan to contributing document. (#1383)
- Closes #1484: Provide default Antibot module config overrides in Quickstart Security module. (#1485)
- Closes #1394: Updating views re-order notice blocks to use render arrays and Url::fromRoute. (#1440)
- #1561: Update composer branch alias for dev-main to 2.4.x-dev. (#1562)
- Closes #1531 Update quick_node_clone to 1.15 for SA-CONTRIB-2022-038 (#1532)
- Fixes #1495: Update Drupal core to 9.2.18 for SA-CORE-2022-008 (#1496)
```
